tests/slack_sdk_async/web/test_web_client_coverage.py

```diff
@@ -15,7 +15,7 @@
 
 
 class TestWebClientCoverage(unittest.TestCase):
-    # 240 endpoints as of Sept 4, 2021
+    # 240 endpoints as of Sept 4, 2021 // TODO :: update before merge
```

Comment:
Since this relies on the publicly accessible methods, not sure how this is to be changed (or not)

Reply:
You can manually add the new elements to `all_api_methods` in the following line of code with a TODO comment saying `# TODO: we will delete this once the APIs are publicly available`.